public/app/features/search/components/SearchItem.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { FC } from 'react';
+import { css, cx } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { e2e } from '@grafana/e2e';
+import { Icon, useTheme, TagList } from '@grafana/ui';
+import appEvents from 'app/core/app_events';
+import { CoreEvents } from 'app/types';
+import { DashboardSectionItem, ItemClickWithEvent } from '../types';
+import { SearchCheckbox } from './SearchCheckbox';
+
+export interface Props {
+  item: DashboardSectionItem;
+  editable?: boolean;
+  onToggleSelection: ItemClickWithEvent;
+  onTagSelected: (name: string) => any;
+}
+
+const { selectors } = e2e.pages.Dashboards;
+
+export const SearchItem: FC<Props> = ({ item, editable, onToggleSelection, onTagSelected }) => {
+  const theme = useTheme();
+  const styles = getResultsItemStyles(theme, item.selected);
+
+  const onItemClick = () => {
+    //Check if one string can be found in the other
+    if (window.location.pathname.includes(item.url) || item.url.includes(window.location.pathname)) {
+      appEvents.emit(CoreEvents.hideDashSearch);
+    }
+  };
+
+  const navigate = () => {
+    window.location.pathname = item.url;
+  };
+
+  const tagSelected = (tag: string, e: React.MouseEvent<HTMLElement>) => {
+    e.stopPropagation();
+    onTagSelected(tag);
+  };
+
+  return (
+    <li aria-label={selectors.dashboards(item.title)} className={styles.wrapper} onClick={navigate}>
+      <SearchCheckbox
+        editable={editable}
+        checked={item.checked}
+        onClick={(e: MouseEvent) => onToggleSelection(item, e)}
+      />
+      <Icon className={styles.icon} name="th-large" />
+      <span className={styles.body} onClick={onItemClick}>
+        <div className={styles.title}>{item.title}</div>
+        <span className={styles.folderTitle}>{item.folderTitle}</span>
+      </span>
+      <TagList tags={item.tags} onClick={tagSelected} className={styles.tags} />
+    </li>
+  );
+};
+
+const getResultsItemStyles = (theme: GrafanaTheme, selected: boolean) => ({
+  wrapper: cx(
+    css`
+      display: flex;
+      align-items: center;
+      margin: ${theme.spacing.xxs};
+      padding: 0 ${theme.spacing.sm};
+      background: ${theme.isLight ? theme.colors.white : theme.colors.dark4};
+      box-shadow: -1px -1px 0 0 hsla(0, 0%, 100%, 0.1),
```

Comment:
Not sure if this and the bg gradient below should be moved to themes.

Reply:
We have some emotion style mixins (pushed an update that uses that)